### PR TITLE
{data}[dummy/dummy] ieeg-cli v1.14.56

### PR DESCRIPTION
**Pull request by murfreesboro**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by murfreesboro** [comment]
Test report by @murfreesboro
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
dev806.vampire - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.5
See https://gist.github.com/4ee5d483741fbb5781b73af18f2ba9de for a full test report.

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
nic17 - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/1f89c89be8bd29dfd39ccaed052fe0d7 for a full test report.

**Comment by smoors** [comment]
Going in, thanks @murfreesboro!

File: easybuild/easyconfigs/i/ieeg-cli/ieeg-cli-1.14.56.eb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+the understanding of epilepsy by providing a platform for sharing data, 
+tools and expertise between researchers."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['https://bitbucket.org/ieeg/ieeg/downloads']
+sources = ['%(name)s-%(version)s-dist.zip']
+checksums = ['fbec0bc36615ffde121e4ca11e04cbc5391003ee557f1596d84c1f53ba9530e8']
+
+dependencies = [('Java', '11.0.2')]
```
Comment (boegel) [inline easybuild/easyconfigs/i/ieeg-cli/ieeg-cli-1.14.56.eb]: @murfreesboro Why not use the `Java/11` wrapper instead, no need to depend on a specific `Java` version?

fixed in #8808 8811

Comment (smoors) [reply easybuild/easyconfigs/i/ieeg-cli/ieeg-cli-1.14.56.eb]: oops, missed that one..

**Pull request merged**